### DISCUSSION TITLE
Custom Logout, Login Message

### DISCUSSION THREAD

**Discussion by norhu1130** [Zrips/CMI#6951] [Q&A]
![image](https://user-images.githubusercontent.com/46096192/172051845-5c5fa53e-36ba-4996-a395-91bdf4bfd08d.png)
Enabled Custom Message, But Not Displayed
![image](https://user-images.githubusercontent.com/46096192/172051888-0cecbcb0-e594-4dbb-b7ca-1354fcf8149f.png)

**Comment by mrfloris** [comment]
What I know about this I've written out here: https://faq.cmi.support/joinleave
hope that helps.
